grpc/rust/BUILD

```diff
@@ -85,3 +86,34 @@ checkstyle_test(
     license_type = "agpl-header",
     size = "small",
 )
+
+cargo_build_script(
+    name = "build_script",
+    srcs = ["build.rs"],
+    data = [
+        "//cluster:proto-files",
+        "//common:proto-files",
+        "//core:proto-files",
```

Comment:
Passing the paths of these targets through `rootpath` did not work, so I left it as it is.